css/style.css

```diff
@@ -1 +1,17 @@
-/* file for your css */
```

Comment:
Here is my CSS for red and green on win or fail:

    .has-success {
        background-color: green;
    }

    .has-fail {
        background-color: red;
    }